src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -1278,6 +1278,81 @@ private Autogenerated.StateItem ToAutogeneratedStateItem(StateTransactionRequest
         }
         #endregion
 
+        #region Distributed Lock API
+        /// <inheritdoc/>
+        [Obsolete]
+        public async override Task<TryLockResponse> TryLock(
```

Comment:
Since this is no longer returning a boolean, it should be `Lock` instead of `TryLock`.